DigitalLearningSolutions.Data/DataServices/CourseDataService.cs

```diff
@@ -21,6 +21,7 @@ public interface ICourseDataService
         IEnumerable<DelegateCourseInfo> GetDelegateCoursesInfo(int delegateId);
         (int totalAttempts, int attemptsPassed) GetDelegateCourseAttemptStats(int delegateId, int customisationId);
         CourseDetails? GetCourseDetails(int customisationId, int centreId, int categoryId);
+        IEnumerable<Course> GetCoursesAtCentreForCategoryId(int centreId, int categoryId);

```

Comment:
Worth double checking if CourseDetails and Course be commonised (I haven't checked these myself)